UDFs (#15075)
cockroachdb/docs: v22.2/sql-statements.md

[`ALTER COLUMN`](alter-column.html) | Change a column's [default constraint](default-value.html), [`NOT NULL` constraint](not-null.html), or [data type](data-types.html).
[`ALTER DATABASE`](alter-database.html) | Apply a schema change to a database.
[`ALTER DEFAULT PRIVILEGES`](alter-default-privileges.html) | Change the default [privileges](security-reference/authorization.html#privileges) for objects created by specific roles/users in the current database.

> [`ALTER FUNCTION`](alter-function.html) | **New in 22.2:** Modify a [user-defined function](user-defined-functions.html).

Reviewer: We still can't use the liquid in the table, right?

```suggestion
[`ALTER FUNCTION`](alter-function.html) | **New in v22.2:** Modify a [user-defined function](user-defined-functions.html).
```

[`COMMENT ON`](comment-on.html) | Associate a comment to a database, table, or column.
[`CONFIGURE ZONE`](configure-zone.html) | Add, modify, reset, or remove a [replication zone](configure-replication-zones.html) for a database, table, index, partition, or system range.
[`CREATE DATABASE`](create-database.html) | Create a new database.

> [`CREATE FUNCTION`](create-function.html) | **New in 22.2:** Create a [user-defined function](user-defined-functions.html).

Reviewer: ```suggestion
[`CREATE FUNCTION`](create-function.html) | **New in v22.2:** Create a [user-defined function](user-defined-functions.html).
```

[`DROP COLUMN`](drop-column.html) | Remove columns from a table.
[`DROP CONSTRAINT`](drop-constraint.html) | Remove constraints from a column.
[`DROP DATABASE`](drop-database.html) | Remove a database and all its objects.

> [`DROP FUNCTION`](drop-function.html) | **New in 22.2:** Drop a [user-defined function](user-defined-functions.html) from a database.

Reviewer: ```suggestion
[`DROP FUNCTION`](drop-function.html) | **New in v22.2:** Remove a [user-defined function](user-defined-functions.html) from a database.
```
